src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -1733,6 +1734,103 @@ <h2 class="text--center">Balance sheet</h2>
               </tr>
               {{end}}
 
+              <!-- BALANCE SHEET - CURRENT ASSETS -->
+
+              {{if $currentAssets := $balanceSheet.current_assets}}
+
+                <tr>
+                  <th class = "strong" id="current-assets-heading">Current assets</th>
+                </tr>
+
+                <!-- BALANCE SHEET - CURRENT ASSETS - STOCKS -->
+
+              {{if $stocks := $currentAssets.stocks}}
+                <tr class="figures">
+                  <td id="stocks-label">Stocks:</td>
```

Comment:
Do we need to match our **id**'s to the **id** in the SmallFullAccountsModifiediXBRL.html?

Ours: `<td id="stocks-label">Stocks:</td>`
SmallFullAccountsModifiediXBRL: `<td id="ac50-label">Stocks:</td>`